addons/website_sale/controllers/main.py

```diff
@@ -268,8 +263,6 @@ def shop(self, page=0, category=None, search='', min_price=0.0, max_price=0.0, p
                 post['tags'] = None
                 tags = {}
 
-        keep = QueryURL('/shop', **self._shop_get_query_url_kwargs(category and int(category), search, min_price, max_price, **post))
-
```

Comment:
Same here, is there any need to touch this line anymore ?

Reply:
Because you have moved the `keep = ...` code higher in the method, before the category was managed ?